### ISSUE TITLE
Downvote reason for question being unrelated to programming or just asking irrelevant question

### ISSUE THREAD

**Issue by Sirmyself** [WillSullivan/idownvotedbecause#48]
some people just think stackoverflow is like a forum. A page explaining what a relevant question is could be great

**Comment by Pamblam** [comment]
If you don't think this violates the [Acceptable subjects](https://github.com/WillSullivan/idownvotedbecause/blob/master/CONTRIBUTING.md#subjects) rule, I can do it.

**Comment by WillSullivan** [comment]
This would be a good example of when it's acceptable. Explaining what is and is not on topic happens all the time. But the range of reasons is pretty broad. I'm not sure if there should be one huge page on the subject, or multiple pages.  

Maybe start off with the simplest case, where the question is blatantly off topic as defined in the help center. Then, we can see if hanging more edge cases feels right, or if other pages should be added as desired.  Feel free to send in a pull request.

**Comment by LindaLawton** [comment]
I think something could be created for Stack overflow but the question would be can you make something general enough to work for all of stack exchange.   As it stands most of these pages could probably be used for all of stack exchange.    @WillSullivan  what is your intention with this?  everything or just stackoverflow?

**Comment by WillSullivan** [comment]
SO is the primary focus, but that's just because that's where I hang, and I got tired explaining why something deserved a downvote. There's no reason why not to branch out. The contributing file details how to add pages for other sites. I'd consider pages that can apply across communities to be put in a subfolder called "stackexchange", or something more general if they apply across domains.

**Comment by klutt** [comment]
I don't see any big reason for this. At least not for Stack Overflow. If a question is off topic, then vote or flag to close it.

**Comment by WillSullivan** [comment]
Yeah, that's the other side of the coin... Can an example of an off topic question being improved (or even the situation as a whole, such as a general misunderstanding of scope) by linking to a page here? I don't really have one, yet. Interested in feedback, tho.
